beacon_node/src/cli.rs

```diff
@@ -1268,15 +1268,5 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .default_value("64")
                 .takes_value(true)
         )
-        .arg(
-            Arg::with_name("disable-duplicate-warn-logs")
```

Comment:
We should probably leave this here as a no-op flag so anyone who has added it doesn't have their BN fail to start